api/app/signals/apps/api/v1/views/csv.py

```diff
@@ -0,0 +1,68 @@
+import glob
+import os
+
+from django.conf import settings
+from django.http import FileResponse
+from django.utils import timezone
+from rest_framework import renderers, status, viewsets
```

Comment:
Just a nit pick, in these cases I like to only import what is needed instead of everything that is in these packages

Reply:
check